mico-core/src/main/java/io/github/ust/mico/core/configuration/ControllerConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Remove license header

Reply:
Why should he remove the license header? We are required to add it. See #432.

Reply:
Ok, didn't know that. But why do we add it then only to this single file and not to all of them?

Reply:
Is the license header somewhere missing? It should be in place in every file.